src/qt/veil/settings/settingsminting.cpp

```diff
@@ -177,35 +187,35 @@ void SettingsMinting::mintzerocoins()
 }
 
 void SettingsMinting::onCheck10Clicked(bool res) {
-    if(res && nPreferredDenom != 10){
+    if(res && !gArgs.IsArgSet("-nautomintdenom") && nPreferredDenom != 10){
```

Comment:
This (and others) shouldn't be necessary since the radio buttons would be disabled if `nautomintdenom` is set.  No sense wasting the cycles for code that wouldn't be reached if it argument was set.

Reply:
Agreed. I'll remove